content/en/blog/falco-0-40.0/index.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Introducing Falco 0.40.0
+date: 2025-01-28
+author: Aldo Lacuku, Federico Di Pierro
+slug: falco-0-40-0
+tags: ["Falco","Release"]
+---
+
+Dear Falco Community, today we are happy to announce the release of Falco 0.40.0!
+
+This version brings several new features, performance improvements, and bug fixes that streamline Falco’s detection capabilities.
+During this release cycle, we merged more than 52 PRs on Falco and more than 150 PRs for libs and drivers, version 0.
+20.0 and version 8.0.0 respectively. Thank you to our maintainers and contributors, as this would not happen without your support and dedication!
```

Comment:
```suggestion
During this release cycle, we merged more than 52 PRs on Falco and more than 150 PRs for libs and drivers, version 0.20.0 and version 8.0.0 respectively. Thank you to our maintainers and contributors, as this would not happen without your support and dedication!
```